cmd/main.go

```diff
@@ -47,6 +52,15 @@ func main() {
 	}
 }
 
+func validateFilters(_ *cobra.Command, _ []string) error {
```

Comment:
It's doing more than validating. It is doing a transformation. Maybe `parseFilters`?